_run/common-kind.mk

```diff
@@ -44,25 +44,25 @@ kind-configure-image:
     cp ./kustomize/akash-node/docker-image.yaml ./kustomize/akash-provider/docker-image.yaml
 
 .PHONY: kind-upload-image
-kind-upload-image:
-	kind --name "$(KIND_NAME)" load docker-image "${DOCKER_IMAGE}"
+kind-upload-image: $(KIND)
+	$(KIND) --name "$(KIND_NAME)" load docker-image "${DOCKER_IMAGE}"
```

Comment:
I want to be able to type `kind`.  I already hate that it's now a pain in the ass to run `akash`.

Maybe we need to re-add an `env.sh` or similar.